src/c++/vdo/tests/RecoveryJournal_t2.c

```diff
@@ -112,6 +112,21 @@ const SectorPattern noSectors[VDO_SECTORS_PER_BLOCK] = {
   { TEAR_OLD, FULL_SECTOR,  GOOD_COUNT, APPLY_NONE },
 };
 
+/**
+ * A non-wrapped journal that has not been corrupted and has no holes.
+ * The reap head is 1 and the highest sequence number is 5.
+ **/
+static BlockPattern basicRecoveryPattern[JOURNAL_BLOCKS] = {
+  { 0, 0, GOOD_COUNT, USE_NONCE, FULL_BLOCK, false, normalSectors },
```

Comment:
I'm probably call this "fullBlocksPattern" or basicFullBlocksPattern" for this. "basic" is a bit generic as a description, and "recovery" is implicit in the test. The pattern itself looks good.

Reply:
Fixed.